### ISSUE TITLE
Extension 'amazonwebservices.aws-toolkit-vscode' CANNOT use API proposal: quickPickSeparators

### ISSUE THREAD

**Issue by zhujik** [aws/aws-toolkit-vscode#2820]
## Problem

Using AWS Toolkit Connect to AWS or Choose Profile command results in the following error:

```
2022-08-10 18:29:21 [ERROR]: aws.login: Error: Extension 'amazonwebservices.aws-toolkit-vscode' CANNOT use API proposal: quickPickSeparators.
Its package.json#enabledApiProposals-property declares: [] but NOT quickPickSeparators.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api amazonwebservices.aws-toolkit-vscode
```

-   OS: Windows 10 running WSL 2.0 (ubuntu)
-   Visual Studio Code version: 1.63.2
-   AWS Toolkit version: 1.47

**Comment by justinmk3** [comment]
> Visual Studio Code version: 1.63.2

Are you able to upgrade VSCode? The latest version is 1.70.

**Comment by justinmk3** [comment]
This API was graduated in vscode 1.64 https://github.com/microsoft/vscode/commit/e15397d2bb4625eb8ab18e69760b8b05e9c4abe7

**Comment by justinmk3** [comment]
Fixed in AWS Toolkit version: 1.48
